### PR TITLE
Build out more functional in plans + xpdan is vagrant

### PR DESCRIPTION
**Pull request by tacaswell**
With this and the pushed state of https://github.com/bluesky/bluesky-pods/pull/31 I can push the raw images from the previous experement through zmq -> xpdan -> zmq -> roi_reduction -> zmq -> echo consumer.

### CODE REVIEW

File: ae_gpcam/bluesky_config/scripts/roi_reduction_consumer.py [hunk 1]
```diff
@@ -131,6 +148,37 @@ def compute_peak_area(Q, I, q_start, q_stop):
     return np.sum((data_section - background) * dQ)
 
 
+def parse_name(inp):
```
Comment (jklynch) [inline ae_gpcam/bluesky_config/scripts/roi_reduction_consumer.py:160]: Could a few example names be added here in a comment?

File: README.md [hunk 2]
```diff
@@ -76,6 +76,19 @@ When you are ready to delete the Vagrant VM use this:
 host:ae-gpcam user$ vagrant destroy
 ```
 
+If when starting up for the first time you get errors implying you can
```
Comment (jklynch) [inline README.md:79]: Would you like to install these packages every time the VM is built?

**Review by jklynch — CHANGES_REQUESTED** [review]

File: ae_gpcam/bluesky_config/scripts/roi_reduction_consumer.py (hunk 1 above)
Comment (tacaswell) [reply ae_gpcam/bluesky_config/scripts/roi_reduction_consumer.py:160]: I would rather not as hopefully we will not have these strings going forward ;)

File: README.md (hunk 2 above)
Comment (tacaswell) [reply README.md:79]: I was seeing these issues with the guest utils not being available before the vagrant file had a chance try and install anything.

I'm very confused by what was going on here (as the missing packages are _inside_ of the guest not on the host).  I assume that this is something to do with getting virtualbox from arch?

**Pull request merged**